app/Http/Controllers/Web/FailedJobController.php

```diff
@@ -67,15 +39,15 @@ public function index()
      */
     public function show($id)
     {
-        $data = \DB::table('failed_jobs')->where('id', '=', $id)->get();
+        $data = DB::table('failed_jobs')->where('id', '=', $id)->get();
```

Comment:
This is making me think we should make a `FailedJob` model, so we can encapsulate some of this database fetching and data parsing logic in there... Not needed for this PR, but maybe a good cleanup task!